src/core/time.ts

```diff
@@ -212,6 +212,48 @@ export class TimeValue {
             }
         } 
     }
+
+    /**
+     * Get a 64 bit binary, little endian representation of this TimeValue.
+     * Used by federates.
+     */
+    public get64Bit() : Buffer {
```

Comment:
```suggestion
    public toBinary() : Buffer {
```

Reply:
This is just for symmetry in the naming (i.e., `fromBinary` and `toBinary`). If you could do the rename that would be great. It can also be `to64Bit` and `from64Bit` -- up to you.